src/operation/run_command/mod.rs

```diff
@@ -64,10 +64,22 @@ impl<'conn> Operation for RunCommand<'conn> {
                 message: "an empty document cannot be passed to a run_command operation".into(),
             })?;
 
+        let mut command = self.command.clone();
```

Comment:
Based on [this language](https://github.com/mongodb/specifications/blob/5334c6c33de34949d39eed913f325ad82dbbcf2a/source/read-write-concern/read-write-concern.rst#generic-command-method-1) from the read/write concern spec, I don't think we need to strip out the empty write concern here. Generally we avoid validation on the documents provided to `run_command` (except for some conditions within a transaction).

Reply:
Removed this in https://github.com/mongodb/mongo-rust-driver/pull/513/commits/6833ab8273d127d549f3fdf105dcd06e7d63fb20.